packages/suite-desktop-core/e2e/fixtures/invity/buy/info.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
These are the mock request responses. Currently not working.
I wonder whether I should keep them here or ditch them completely? What do you think?